core/semaphore.cc

```diff
@@ -69,8 +69,8 @@ bool semaphore::trywait(unsigned units)
     }
 
     return ok;
-}
 
 
 
 
+}
```

Comment:
Please remove this part of the patch (you aren't touching semaphore.cc code at all, so no need to move the brace)